cms/settings/base.py

```diff
@@ -196,38 +197,51 @@
 #
 # Do not use the same Redis instance for other things like Celery!
 
-# Prefer the TLS connection URL over non
-REDIS_URL = env.get("REDIS_TLS_URL", env.get("REDIS_URL"))
+CACHES: dict = {"memory": {"BACKEND": "django.core.cache.backends.locmem.LocMemCache", "LOCATION": "memory"}}
```

Comment:
suggestion:

```suggestion
# The memory cache will be used by ElastiCacheIAMCredentialProvider when configured
CACHES: dict = {"memory": {"BACKEND": "django.core.cache.backends.locmem.LocMemCache", "LOCATION": "memory"}}
```

or something along those lines.

Reply:
I don't think that's useful. The memory cache can be used by anything which needs it (sure, it might not be that many things in reality), but putting the comment there means it's more likely to get out of date.

Reply:
I suppose this will be mitigated when we have the tests for the new credential provider. What I want to avoid is someone removing it down the line only to discover it explodes

Reply:
Yep let's solve this with tests instead!